content/blog/2020-05-04-covid-19-open-data/index.md

```diff
@@ -0,0 +1,126 @@
+---
+slug: "covid-19-open-data"
+title: rOpenSci tools to access a world of open data for your research project
+author:
+  - Maëlle Salmon
+  - Stefanie Butland
+  - Karthik Ram
+  - Noam Ross
+  - Mark Padgham
+  - Jeroen Ooms
+  - Carl Boettiger
+  - Scott Chamberlain
+date: 2020-05-04
+tags:
+  - packages
+  - R
+  - community
+  - reproducibility
+  - data
+  - data-access
+  - open-data
+description: "Covid-19 and Open Data from rOpenSci."
+# twitterImg: blog/2019/06/04/post-template/name-of-image.png
+---
+
+The Covid-19 pandemic has changed all of our lives. For many students, the lab or field research that they had planned for their dissertation may not be possible right now, stalling progress of their studies. However, if you have internet service, it may still be possible to conduct scientific research based on the massive amounts of open data that are available.
```

Comment:
> "if you have internet service"

I'd remove that, since a bit more is needed (programming skills, even if basic).